src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -0,0 +1,599 @@
+    console.log(`${bufferedSeconds} seconds of video are ready to play.`);
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, the browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver][datasaver] is enabled, Chrome forces the `preload` value to
+  `none`.
+- On a cellular connection (2G, 3G, and 4G), Chrome forces the `preload` value to
+  `metadata`.
+
+### Tips
+
+If your website contains many video resources on the same domain, I would
+recommend you set the `preload` value to `metadata` or define the `poster`
+attribute and set `preload` to `none`. That way, you would avoid hitting
+the maximum number of HTTP connections to the same domain (6 according to the
+HTTP 1.1 spec) which can hang loading of resources. Note that this may also
+improve page speed if videos aren't part of your core user experience.
+
+## Link preload
+
+As [covered] in other [articles], [link preload] is a declarative fetch that
+allows you to force the browser to make a request for a resource without
+blocking the `window.onload` event and while the page is downloading. Resources
+loaded via `<link rel="preload">` are stored locally in the browser, and are
+effectively inert until they're explicitly referenced in the DOM, JavaScript,
+or CSS.
+
+Preload is different from prefetch in that it focuses on current navigation and
+fetches resources with priority based on their type (script, style, font,
+video, audio, etc.). It should be used to warm up the browser cache for current
+sessions.
+
+<figure>
+  <img src="./link-preload.svg">
+</figure>
+
+### Preload full video
+
+Here's how to preload a full video on your website so that when your
+JavaScript asks to fetch video content, it is read from cache as the resource
+may have already been cached by the browser. If the preload request hasn't
+finished yet, a regular network fetch will happen.
+
+```js
+<link rel="preload" as="video" href="https://cdn.com/small-file.mp4">
+
+<video id="video" controls></video>
+
+<script>
+  // Later on, after some condition has been met, set video source to the
+  // preloaded video URL.
+  video.src = 'https://cdn.com/small-file.mp4';
+  video.play().then(_ => {
+    // If preloaded video URL was already cached, playback started immediately.
+  });
+</script>
+```
+
+{% Aside %}
+  I would recommend using this only for small media files (less than 5MB).
+{% endAside %}
+
+Because the preloaded resource is going to be consumed by a video element in
+the example, the `as` preload link value is `video`. If it were an audio
+element, it would be `as="audio"`.
+
+### Preload the first segment
+
+The example below shows how to preload the first segment of a video with `<link
+rel="preload">` and use it with Media Source Extensions. If you're not familiar
+with the MSE Javascript API, please read [MSE basics][mse-basics].
```

Comment:
```suggestion
with the MSE Javascript API, see [MSE basics][mse-basics].
```
https://developers.google.com/style/tone#politeness-and-use-of-please

Reply:
@jpmedley this MSE basics seems like another candidate to migrate over to your media collection?